src/sb-util.ts

```diff
@@ -221,43 +226,78 @@ export class Sprite extends SpriteCollection {
     /**
      * @return {VariableCollection}. The variables available to a Sprite include the
      *      variables attached to the sprite as well as variables in the
-     *      global scope, which are attached to the stage
+     *      global scope, which are attached to the stage. Also included in variables
+     *      are broadcasts (which are always global) and lists, which can be global
+     *      or local.
      */
     public variables(): VariableCollection {
         let taggedVariables: Iterable<VariableProperties>;
 
+        function setMetaSprite(
+            deserialized: Iterable<VariableProperties>,
+            sprite: SpriteProperties,
+        ): Iterable<VariableProperties> {
+            return setMetaIterable(
+                deserialized,
+                (v): VariableProperties => {
+                    getVariableMeta(v).sprite = sprite;
+                    return v;
+                },
+            );
+        }
+
```

Comment:
I'm curious if defining `setMetaSprite` as an inline function inside `variables` is intentional? Is there any value in having it as a private function outside `variables`?

Reply:
It is probably a copy of how I did the first pass for the others - I think this will work for now but we should come up with a better `setMetaPropsWhileIterating` pattern :)